### ISSUE TITLE
CC0030: Bad grammar in message

### ISSUE THREAD

**Issue by davidhesselbom** [code-cracker/code-cracker#838]
# Bug

This is very minor, but, in version 1.0.1, the message description for CC0030 is "This variables can be made const." for the following piece of code:

``` csharp
static void Main(string[] args)
{
    var s = "Hello";
    var t = "World";
    Console.WriteLine(s + " " + t);
}
```

For the following piece of code, I get two of these messages, one for `s` and one for `t`:

``` csharp
static void Main(string[] args)
{
    var s = "Hello";

    var t = "World";

    Console.WriteLine(s + " " + t);
}
```

The message should either be 
- "These variables can be made const." in the former case and "This variable can be made const." in the latter, or 
- "This variable/These variables can be made const.", if figuring out whether one or several variables triggered the message is too much work. 

Just... don't let it say "This variables".

**Comment by jwooley** [comment]
There are actually a number of messages that should be reviewed for grammatical correctness. I would recommend a task item to review the values in the resource file in general since all of the changes should be made there (or if there are any strings still lingering in individual analyzers, moved to the resource file).

**Comment by giggio** [comment]
Yes, we have those errors. Most contributors are not native English speakers. Thanks for pointing this out.

**Comment by giggio** [comment]
@jwooley Good idea, created #839.

**Comment by davidhesselbom** [comment]
@giggio Yes, I noticed after using the extension for a while longer. This particular message stood out at first, probably because I got that same message a lot :) #839 doesn't seem to mention what @jwooley brought up about strings still lingering in individual analyzers. I don't know how the source code is organized, but I found two files with the string "this variables", one of which is a test file:

[MakeLocalVariableConstWhenItIsPossibleAnalyzer.cs](https://github.com/code-cracker/code-cracker/blob/master/src/CSharp/CodeCracker/Performance/MakeLocalVariableConstWhenItIsPossibleAnalyzer.cs)

[MakeLocalVariablesConstWhenItIsPossibleTests.cs](https://github.com/code-cracker/code-cracker/blob/master/test/CSharp/CodeCracker.Test/Performance/MakeLocalVariablesConstWhenItIsPossibleTests.cs)

I also found a corresponding and similarly, but not identically, named file that doesn't have the bad grammar:

[MakeLocalVariableConstWhenPossibleAnalyzer.vb](https://github.com/code-cracker/code-cracker/blob/master/src/VisualBasic/CodeCracker/Performance/MakeLocalVariableConstWhenPossibleAnalyzer.vb)

It looks to me like #360 can fix inconsistencies like this, and that it would good idea to complete #360 before starting to work on #839.
